checkov/common/bridgecrew/platform_integration.py

```diff
@@ -422,9 +423,24 @@ def persist_enriched_secrets(self, enriched_secrets: list[EnrichedSecret]) -> st
                           f' repo_path={self.repo_path}, bucket={self.bucket}')
             return None
 
-        repo_path_without_src = os.path.dirname(self.repo_path)
-        s3_path = f'{repo_path_without_src}/{checkov_results_prefix}/{CheckType.SECRETS}/secrets_to_verify.json'
-        _put_json_object(self.s3_client, enriched_secrets, self.bucket, s3_path)
+        if not bc_integration.bc_api_key or not os.getenv("CKV_VALIDATE_SECRETS"):
+            logging.debug('Skipping persistence of enriched secrets object as secrets verification is off,'
+                          ' enabled it via env var CKV_VALIDATE_SECRETS and provide an api key')
+            return None
+
+        base_path = re.sub(r'checkov/(.*?)/src', r'original_secrets/\1', self.repo_path)
```

Comment:
can you just move it `r'checkov/(.*?)/src'` up to global with a `re.compile()` then linters can also pick it up better